csv.go

```diff
@@ -0,0 +1,180 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/davecgh/go-spew/spew"
```

Comment:
I am pretty sure you can achieve the same thing without using an external library. We definitely do not need a dependency that is abandoned 7 years ago.